pkg/sql/logictest/testdata/logic_test/create_index_on_system_table

```diff
@@ -0,0 +1,9 @@
+statement ok
+INSERT INTO system.users VALUES ('node', '', false);
+GRANT node TO root;
+
+# Any system table other than system.descriptor and system.zones
```

Comment:
Why can't `system.descriptor` or `system.zones` be split across multiple ranges?